pkg/terminal/command.go

```diff
@@ -864,6 +869,18 @@ func parseArgs(args string) ([]string, error) {
 	return v[0], nil
 }
 
+// parseOptionalCount parses an optional count argument.
+// If there are not arguments, a value of 1 is returned as the default.
+func parseOptionalCount(arg string) (val int64, err error) {
+	if len(arg) == 0 {
+		return 1, nil
+	}
+	if val, err = strconv.ParseInt(arg, 0, 64); err != nil {
```

Comment:
This was marked as resolved but I think @aarzilli comment still makes sense. We ignore the result of this if `err != nil` so just return the result directly.

Reply:
sorry, got lost. I resolved this comment (and the one about the default value), but only actually made the change on one. Sorry, thanks!